### DISCUSSION TITLE
OIDC connector on OSS version

### DISCUSSION THREAD

**Discussion by ajagnanan** [gravitational/teleport#9151] [General]
Is it possible to add the OIDC connector to the free version instead of the Github connector? This will make it more flexible and can even work with Github if the user so desires. Thanks.

**Comment by darius-m** [comment]
It is possible that this a feature was added later, and has become part of the enterprise version as more authentication methods were implemented, but it is strange to not be able to connect other open source tools, like Keycloak, as identity providers.

**Comment by zen** [comment]
I guess it's part of Gravitational business model.
